doc/source/user_guide/scale.rst

```diff
@@ -219,188 +219,10 @@ different library that implements these out-of-core algorithms for you.
-
-   divisions = tuple(pd.to_datetime(starts)) + (pd.Timestamp(ends[-1]),)
-   ddf.divisions = divisions
-   ddf
-
-Now we can do things like fast random access with ``.loc``.
-
-.. ipython:: python
-   :okwarning:
-
-   ddf.loc["2002-01-01 12:01":"2002-01-01 12:05"].compute()
-
-Dask knows to just look in the 3rd partition for selecting values in 2002. It
-doesn't need to look at any other data.
-
-Many workflows involve a large amount of data and processing it in a way that
-reduces the size to something that fits in memory. In this case, we'll resample
-to daily frequency and take the mean. Once we've taken the mean, we know the
-results will fit in memory, so we can safely call ``compute`` without running
-out of memory. At that point it's just a regular pandas object.
-
-.. ipython:: python
-   :okwarning:
-
-   @savefig dask_resample.png
-   ddf[["x", "y"]].resample("1D").mean().cumsum().compute().plot()
-
-.. ipython:: python
-   :suppress:
-
-   import shutil
-
-   shutil.rmtree("data/timeseries")
-
-These Dask examples have all be done using multiple processes on a single
-machine. Dask can be `deployed on a cluster
-<https://docs.dask.org/en/latest/setup.html>`_ to scale up to even larger
-datasets.
-
-You see more dask examples at https://examples.dask.org.
-
-Use Modin
----------
-
-Modin_ is a scalable dataframe library, which aims to be a drop-in replacement API for pandas and
-provides the ability to scale pandas workflows across nodes and CPUs available. It is also able
-to work with larger than memory datasets. To start working with Modin you just need
-to replace a single line of code, namely, the import statement.
-
-.. code-block:: ipython
-
-   # import pandas as pd
-   import modin.pandas as pd
-
-After you have changed the import statement, you can proceed using the well-known pandas API
-to scale computation. Modin distributes computation across nodes and CPUs available utilizing
-an execution engine it runs on. At the time of Modin 0.27.0 the following execution engines are supported
-in Modin: Ray_, Dask_, `MPI through unidist`_, HDK_. The partitioning schema of a Modin DataFrame partitions it
-along both columns and rows because it gives Modin flexibility and scalability in both the number of columns and
-the number of rows.
-
-For more information refer to `Modin's documentation`_ or the `Modin's tutorials`_.
-
-.. _Modin: https://github.com/modin-project/modin
-.. _`Modin's documentation`: https://modin.readthedocs.io/en/latest
-.. _`Modin's tutorials`: https://github.com/modin-project/modin/tree/master/examples/tutorial/jupyter/execution
-.. _Ray: https://github.com/ray-project/ray
-.. _Dask: https://dask.org
-.. _`MPI through unidist`: https://github.com/modin-project/unidist
-.. _HDK: https://github.com/intel-ai/hdk
-.. _dask.dataframe: https://docs.dask.org/en/latest/dataframe.html
+There are many other libraries which provide similar APIs to pandas and work nicely with pandas DataFrame,
```

Comment:
```suggestion
There are other libraries which provide similar APIs to pandas and work nicely with pandas DataFrame,
```

Reply:
Thank you for reviewing and giving me suggestions @datapythonista . I saw the unit test ASAN/UBSAN failed, but is it okay?

Reply:
Yes, our CI seems to be failing now, you can ignore that failure.